assets/app/view/welcome.rb

```diff
@@ -20,6 +20,7 @@ def render_notification
 
         <p>18Carolinas, 18NewEngland and 18NewEngland2 are now in production.</p>
         <p><a href='https://github.com/tobymao/18xx/wiki/18USA'>18USA</a> is in beta.</p>
+        <p><a href='https://github.com/tobymao/18xx/wiki/1822MX'>1822MX</a> is in alpha.</p>
```

Comment:
Put this first and remove the 18USA line

Reply:
Changed

Reply:
Done